tests/acceptance/SubscriberManageImportExportCest.php

```diff
@@ -101,6 +101,21 @@ private function chooseListAndConfirm(\AcceptanceTester $i) {
     $i->waitForText('Import again');
   }
 
+  private function createNewListAndConfirm(\AcceptanceTester $i) {
+    $newListName = 'Simple List';
+    // first create a new list
+    $i->click('a.mailpoet_create_segment');
```

Comment:
Can you use $i->click(`Create a new list`) here? It is closer to user's context.

Reply:
Sure, I just changed to your solution. Thanks.